weTranslate/NoResultViewModel.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  NoResultViewModel.swift
+//  weTranslate
+//
+//  Created by Lionel on 2/28/16.
+//  Copyright © 2016 weTranslate. All rights reserved.
+//
+
+import Foundation
+import TranslateKit
+
+struct NoResultViewModel {
+
+    let word: String
+    let fromLanguage: Language
+    let toLanguage: Language
+
+    init(word: String, fromLanguage: Language, toLanguage: Language) {
```

Comment:
An issue has been created for that : https://github.com/jenniferlim/weTranslate/issues/40